### DISCUSSION TITLE
Book seems empty

### DISCUSSION THREAD

**Discussion by Sytten** [time-rs/time#483] [Question] [answered]
Hi!

I am trying to move from chrono to time so I went to the book to have beginner helps, but it seems empty?

Thanks!

**Comment by jhpratt** [comment, marked as answer]
#327

If you have any specific questions about migrating, feel free to ask here and I'll do my best to answer.

**Reply by Sytten** [reply]
I don't have anything specific, just trying to find equivalent for super basic stuff like `DateTime::<Utc>` and getting to know the library.
